src/filesystem.tsx

```diff
@@ -64,12 +64,11 @@ export async function runPDFIngestion() {
   return response;
 }
 
-export async function readFile(file: FileEntry) {
-  const content = await readBinaryFile(file.path);
-  return content;
+export async function readEntryFileAsBinary(file: FileEntry) {
```

Comment:
nit: Shouldn't it be `readFileEntryAsBinary` instead of `ReadEntryFileAsBinary`?

Reply:
Will fix this now.